specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9093 @@
+            "AzureDataExplorer",
+            "AzureFileService",
+            "AzureSqlDatabase",
+            "AmazonPostgreSql",
+            "AzurePostgreSql",
+            "SqlServerDatabase",
+            "AzureSqlDatabaseManagedInstance",
+            "AzureSqlDataWarehouse",
+            "AzureMySql",
+            "AzureStorage",
+            "Teradata",
+            "Oracle",
+            "SapS4Hana",
+            "SapEcc",
+            "PowerBI"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DataSourceType",
+            "modelAsString": true
+          }
+        },
+        "scans": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Scan"
+          },
+          "readOnly": true
+        }
+      },
+      "discriminator": "kind"
+    },
+    "DataSourceList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataSource"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "FilterProperties": {
+      "required": [
+        "excludeUriPrefixes",
+        "includeUriPrefixes"
+      ],
+      "type": "object",
+      "properties": {
+        "excludeUriPrefixes": {
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "includeUriPrefixes": {
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "excludeRegexes": {
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "includeRegexes": {
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Filter": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/FilterProperties"
+            }
+          ]
+        }
+      }
+    },
+    "IntegrationRuntime": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "SelfHosted",
+            "Managed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IntegrationRuntimeType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "IntegrationRuntimeStatus": {
```

Comment:
IntegrationRuntimeStatus contains 'kind' only, but no states like 'running', 'failed', 'limited' ？

Reply:
Had offline conversation, Status field comes from specific kind of IntegrationRuntimeStatus